.gitignore

```diff
@@ -17,5 +17,5 @@ _build
 *.iml
 rebar3.crashdump
 .DS_Store
-.env*
+*.env
```

Comment:
Please leave the `.env*` in place too.. a few of us use dotenv to set up those environment variables on the fly in the shell